packages/utils/cli/templates/stories/[Component.stories.tsx].js

```diff
@@ -0,0 +1,16 @@
+import { pascalCase } from 'pascal-case'
```

Comment:
We don't need the `stories` folder anymore 🧹 🥳

Reply:
Indeed, package structure is now documented here and everything is flat inside the `src` folder: https://sparkui.vercel.app/?path=/docs/definitions-package-structure--docs

Reply:
Nice!! I see that it's working fine right now 🥳